tldw_Server_API/app/api/v1/endpoints/audiobooks.py

```
@@ -0,0 +1,453 @@
+            return project_id
+    return f"abk_{job_row.get('id', 0)}"
+
+
+@router.post(
+    "/parse",
+    response_model=AudiobookParseResponse,
+    summary="Parse audiobook source and detect chapters",
+)
+async def parse_audiobook_source(
+    request: AudiobookParseRequest,
+    _current_user: User = Depends(get_request_user),
```

Ruff (0.14.13)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)